openstack/sap-seeds/templates/_flavors_deleted.tpl

```diff
@@ -33,9 +52,30 @@
   extra_specs:
 {{- if .Values.use_hana_exclusive }}
     {{- tuple . "vmware_hana_exclusive" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
-    "trait:CUSTOM_NUMASIZE_C48_M729": "required"
 {{- else }}
     {{- tuple . "vmware_common" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
     "resources:CUSTOM_BIGVM": "2"
     "host_fraction": "1,1/2"
 {{- end }}
+    "trait:CUSTOM_NUMASIZE_C48_M729": "required"
+    "hw:cpu_cores": "48"  # used in nova-vmware as cores-per-socket (24pCPU = 48vCPU)
+    "vmware:hw_version": "vmx-18"
+
+- name: "x1.32xlarge"
+  id: "250"
+  vcpus: 128
+  ram: 1991680
+  disk: 64
+  is_public: false
+  extra_specs:
+{{- if .Values.use_hana_exclusive }}
+    {{- tuple . "vmware_hana_exclusive" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+{{- else }}
+    {{- tuple . "vmware_common" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+    "host_fraction": "1,0.67,0.34"
+    "resources:CUSTOM_BIGVM": "2"
+    "resources:CUSTOM_MEMORY_RESERVABLE_MB": "1991680"
+{{- end }}
+    "trait:CUSTOM_NUMASIZE_C48_M729": "required"
+    "hw:cpu_cores": "48"  # used in nova-vmware as cores-per-socket (24pCPU = 48vCPU)
```

Comment:
I noticed that maybe a bit late, but it says here `hw:cpu_cores=48` vs `vcpus=128` in line 66. Is that intended?

Reply:
As the comment implies, this number is the cores-per-socket, a.k.a. the vCPU NUMA size for the flavor. So it's always <= the number of vCPUs of the flavor.

Reply:
this is highly confusing in nova -- i thought the comment would clear it up, but since other people have been confused by it too already, even with the comment, i should probably reword it to make it clearer. -> separate PR

Reply:
So it is intended. I still surprised that it is a non-integer factor.